CMakeLists.txt

```diff
@@ -0,0 +1,315 @@
+###############################################################################
+## Copyright (C) 2009-2024 Arzel Jérôme <myst6re@gmail.com>
+## Copyright (C) 2020 Julian Xhokaxhiu <https://julianxhokaxhiu.com>
+##
+## This program is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## This program is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with this program.  If not, see <http://www.gnu.org/licenses/>.
+###############################################################################
+
+cmake_minimum_required(VERSION 3.25)
+cmake_policy(SET CMP0010 NEW)
+cmake_policy(SET CMP0042 NEW)
+cmake_policy(SET CMP0074 NEW)
+cmake_policy(SET CMP0091 NEW)
+
+set(VCPKG_DISABLE_COMPILER_TRACKING 1)
+set(VCPKG_INSTALL_OPTIONS "--clean-after-build")
+set(X_VCPKG_APPLOCAL_DEPS_INSTALL ON)
+
+set(RELEASE_NAME "Hyne")
+set(GUI_TARGET "${RELEASE_NAME}")
+if(NOT PRERELEASE_STRING)
+    set(PRERELEASE_STRING "")
+endif()
+
+project("Hyne" VERSION 1.0.0 LANGUAGES CXX
+    DESCRIPTION "Final Fantasy VIII savegame editor"
+    HOMEPAGE_URL "https://github.com/myst6re/hyne"
+)
+
+# Try to autodetect Qt path if not specified
+include(qt.cmake)
+
+# specify the C++ standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+if (MSVC)
+    add_compile_options(/W3)
+else()
+    add_compile_options(-Wall -Wextra -pedantic)
+endif()
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_AUTOUIC OFF)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+option(GUI "Build the gui executable" ON)
```

Comment:
This option is not needed for this project, can you remove it please?

Reply:
If you prefer you can set it as a variable, not an option

Reply:
Fair enough, let me see how hard it would be to get rid of it

Reply:
Allright, done and CI seems to pass :) https://github.com/julianxhokaxhiu/hyne/actions/runs/16549550665